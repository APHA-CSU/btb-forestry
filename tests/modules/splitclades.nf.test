nextflow_process {

    name "Test Process splitclades"
    script "main.nf"
    process "SPLIT_CLADES"

    test("Should split clades") {

        when {
            params {
            }
            process {
                """
                input[0] = file("${projectDir}/tests/data/Allclean_exp.csv")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
