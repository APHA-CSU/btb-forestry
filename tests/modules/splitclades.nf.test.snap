{
    "Should split clades": {
        "content": [
            {
                "0": [
                    "B6-16_Pass.csv:md5,fc11ccb87549416a479bb45383f2014d"
                ],
                "passSamples": [
                    "B6-16_Pass.csv:md5,fc11ccb87549416a479bb45383f2014d"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-08-28T07:46:27.310693794"
    }
}